Titanium.Web.Proxy/CertificateHandler.cs

```diff
@@ -108,17 +100,8 @@ public partial class ProxyServer
                     ClientCertificate = clientCertificate
                 };
 
-
-                Delegate[] invocationList = ClientCertificateSelectionCallback.GetInvocationList();
-                Task[] handlerTasks = new Task[invocationList.Length];
-
-                for (int i = 0; i < invocationList.Length; i++)
-                {
-                    handlerTasks[i] = ((Func<object, CertificateSelectionEventArgs, Task>) invocationList[i])(null, args);
-                }
-
-                Task.WhenAll(handlerTasks).Wait();
-
+                //why is the sender null?
+                ClientCertificateSelectionCallback.InvokeParallel(null, args);
```

Comment:
Why is the sender null here? It can be the sender parameter of the calling method (which is an SslStream) or the ProxyServer object. (+Same question in ServerCertificateSelectionCallback)